### PR TITLE
AST-1350 - E2E test case for social icon background color

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for social icon background color

### How has this been tested?

Run single test : npm run test:e2e:interactive tests/e2e/specs/customizer/social-icons/social-icon-background-color.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-color.test.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Social Icons in the customizer', () => {
+	it( 'social icon background color for desktop should apply correctly', async () => {
+		const socialiconbackColor = {
+			'header-social-1-bg-color': {
+				desktop: 'rgb(198, 68, 47)',
```
Comment (premanshup) [inline tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-color.test.js:7]: @SimranChougule1 Any reason have not added background color and test cases for tablet and mobile devices?

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-color.test.js]: added test case for tablet and mobile it takes same color please review

File: tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-color.test.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+import { setBrowserViewport } from '../../../../utils/set-browser-viewport';
+describe( 'Social Icons in the customizer', () => {
+	it( 'social icon background color should apply correctly', async () => {
+		const BackgroundColor = {
```
Comment (GauravKhupse) [inline tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-color.test.js:6]: THe Camecase is needed for a constant variable like'**backGroundColor**'

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/social-icons/social-icon-background-color.test.js]: Done

**Review by GauravKhupse — APPROVED** [review]

**Review by bsfsnehal — APPROVED** [review]

**Pull request merged**